### PR TITLE
[17.0][MIG] fieldservice_activity

### PR DESCRIPTION
**Pull request by Deriman-Alonso**
Standart migration.

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

**Review by DorianMAG — APPROVED** [review]
Hi
Code review, it looks good.
Thx for the work

**Comment by DorianMAG** [comment]
@Deriman-Alonso or @max3903 
Possible to rebase plz.
Runboat doesn't work

**Comment by Deriman-Alonso** [comment]
@DorianMAG Pre-commit changed some stuff in intex.html, hopefully runboat will run now.

**Review by JulienMartinez — APPROVED** [review]
Hi
LGTM
thx

**Comment by DorianMAG** [comment]
> @DorianMAG Pre-commit changed some stuff in intex.html, hopefully runboat will run now.

Thx,
perfect was work :)

**Review by DorianMAG — CHANGES_REQUESTED** [review]
Functional testing failed. 
Unable to edit read-only fields conditioned by expression. Syntax changed in v17. 

https://github.com/OCA/field-service/pull/1183/files#diff-e7e26d6a3ed185daa05317dcf7ed3457ff1831d1fb55c788d669372c2d38fd4dR12

The XML syntax is as follows: 
https://www.odoo.com/documentation/17.0/developer/reference/user_interface/view_architectures/field_attribute_readonly.html

but I have not found the syntax for declaring fields in Python.

**Comment by Deriman-Alonso** [comment]
I will  look into it, Ty for review.

File: fieldservice_activity/views/fsm_order.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                            <field name="ref" />
+                            <field name="completed" column_invisible="1" />
+                            <field name="completed_on" />
+                            <field name="completed_by" />
+                            <button
+                                name="action_done"
+                                string="Complete"
+                                type="object"
+                                class="oe_highlight"
+                                invisible="('state', 'not in', ['todo'])"
```
Comment (DorianMAG) [inline fieldservice_activity/views/fsm_order.xml:24]: Same here, the syntax change for:

invisible="state not in ['todo']"

**Review by DorianMAG — CHANGES_REQUESTED** [review]

**Comment by dreispt** [comment]
Superseded by #1211

**Pull request closed without merging**